lib/collection/item.js

```diff
@@ -216,17 +230,37 @@ _.assign(Item.prototype, /** @lends Item.prototype */ {
     },
 
     /**
-     *  Set or update protocol profile behavior for the current Item
+     * Set, update or delete protocol profile behavior for the current Item.
+     *
+     * @example <caption> Set or update protocol profile behavior </caption>
+     * item.setProtocolProfileBehavior({ key: 'strictSSL', value: false });
+     *
+     * @example <caption> Delete protocol profile behavior </caption>
+     * // missing property value signifies delete
+     * item.setProtocolProfileBehavior({ key: 'strictSSL' });
      *
      * @private
-     * @param {String} key
-     * @param {*} value
+     * @param {Object} obj
+     * @param {String} obj.key
```

Comment:
Descriptions to be updated depending on the signature we choose (It should accept a `key` and `value` argument instead of an object).